src/Symfony/Component/Cache/Tests/Adapter/FilesystemTest.php

```diff
@@ -27,4 +27,28 @@ public function createCachePool()
 
         return new FilesystemAdapter('sf-cache');
     }
+
+    public static function tearDownAfterClass()
+    {
+        self::rmdir(sys_get_temp_dir().'/symfony-cache');
+    }
+
+    public static function rmdir($dir)
+    {
+        if (!$dir || 0 !== strpos(dirname($dir), sys_get_temp_dir())) {
+            throw new \Exception(__METHOD__."() operates only on subdirs of system's temp dir");
+        }
+        $children = new \RecursiveIteratorIterator(
+            new \RecursiveDirectoryIterator($dir, \RecursiveDirectoryIterator::SKIP_DOTS),
+            \RecursiveIteratorIterator::CHILD_FIRST
+        );
+        foreach($children as $child) {
```

Comment:
@nicolas-grekas, I've installed the latest release of `fabpot/php-cs-fixer` from the `1.x` branch (`v1.11.2`); and it seems to be working correctly. Do you know which version is using fabbot.io?
Indeed, please note that there are other issues detected too (including rename of `FilesystemAdapterTest` by `FilesystemTest` according PSR-0):
```
$ composer show -i fabpot/php-cs-fixer
You are running composer with xdebug enabled. This has a major impact on runtime performance. See https://getcomposer.org/xdebug
You are using the deprecated option "installed". Only installed packages are shown by default now. The --all option can be used to show all packages.
name     : fabpot/php-cs-fixer
descrip. : A tool to automatically fix PHP code style
keywords : 
versions : * v1.11.2
type     : application
license  : MIT License (MIT) (OSI approved) https://spdx.org/licenses/MIT.html#licenseText
source   : [git] https://github.com/FriendsOfPHP/PHP-CS-Fixer.git 41f70154642ec0f9ea9ea9c290943f3b5dfa76fc
dist     : [zip] https://api.github.com/repos/FriendsOfPHP/PHP-CS-Fixer/zipball/41f70154642ec0f9ea9ea9c290943f3b5dfa76fc 41f70154642ec0f9ea9ea9c290943f3b5dfa76fc
names    : fabpot/php-cs-fixer

autoload
psr-4
Symfony\CS\ => Symfony/CS/

requires
ext-tokenizer *
php >=5.3.6
sebastian/diff ~1.1
symfony/console ~2.3|~3.0
symfony/event-dispatcher ~2.1|~3.0
symfony/filesystem ~2.1|~3.0
symfony/finder ~2.1|~3.0
symfony/process ~2.3|~3.0
symfony/stopwatch ~2.5|~3.0

requires (dev)
satooshi/php-coveralls 0.7.*@dev
```
```php
<?php
// .php.cs
return Symfony\CS\Config\Config::create()
    ->setUsingLinter(false)
    ->setUsingCache(true)
    ->finder(
        Symfony\CS\Finder\DefaultFinder::create()
            ->in(__DIR__)
    )
;
```
```
$ ./vendor/bin/php-cs-fixer fix -vvv
Loaded config from "/home/phansys/symfony/src/Symfony/Component/Cache/.php_cs"
You are running php-cs-fixer with xdebug enabled. This has a major impact on runtime performance.
......F.....F..........
Legend: ?-unknown, I-invalid file syntax, file ignored, .-no changes, F-fixed, E-error
   1) Tests/Adapter/FilesystemTest.php (psr0, braces)
   2) Tests/Adapter/ApcuAdapterTest.php (function_declaration)
Fixing time per file:
[0.147 s] DoctrineProvider.php
[0.296 s] CacheItem.php
[0.022 s] Exception/InvalidArgumentException.php
[0.023 s] Exception/CacheException.php
[0.142 s] Tests/Fixtures/ExternalAdapter.php
[0.121 s] Tests/DoctrineProviderTest.php
[0.126 s] Tests/Adapter/FilesystemTest.php
[0.104 s] Tests/Adapter/ChainAdapterTest.php
[0.056 s] Tests/Adapter/DoctrineAdapterTest.php
[0.120 s] Tests/Adapter/ProxyAdapterTest.php
[0.044 s] Tests/Adapter/NamespacedProxyAdapterTest.php
[0.100 s] Tests/Adapter/RedisAdapterTest.php
[0.124 s] Tests/Adapter/ApcuAdapterTest.php
[0.064 s] Tests/Adapter/ArrayAdapterTest.php
[0.188 s] Adapter/ApcuAdapter.php
[0.518 s] Adapter/ProxyAdapter.php
[0.268 s] Adapter/RedisAdapter.php
[0.555 s] Adapter/ArrayAdapter.php
[0.024 s] Adapter/AdapterInterface.php
[0.523 s] Adapter/ChainAdapter.php
[0.483 s] Adapter/FilesystemAdapter.php
[0.143 s] Adapter/DoctrineAdapter.php
[1.101 s] Adapter/AbstractAdapter.php

Fixed all files in 5.298 seconds, 8.000 MB memory used
```
```
$ git diff
diff --git a/src/Symfony/Component/Cache/Tests/Adapter/ApcuAdapterTest.php b/src/Symfony/Component/Cache/Tests/Adapter/ApcuAdapterTest.php
index 8b6493b..5e510e7 100644
--- a/src/Symfony/Component/Cache/Tests/Adapter/ApcuAdapterTest.php
+++ b/src/Symfony/Component/Cache/Tests/Adapter/ApcuAdapterTest.php
@@ -36,7 +36,7 @@ class ApcuAdapterTest extends CachePoolTest
         $pool = $this->createCachePool();
 
         $item = $pool->getItem('foo');
-        $item->set(function() {});
+        $item->set(function () {});
 
         $this->assertFalse($pool->save($item));
 
diff --git a/src/Symfony/Component/Cache/Tests/Adapter/FilesystemTest.php b/src/Symfony/Component/Cache/Tests/Adapter/FilesystemTest.php
index 2820012..5c566a1 100644
--- a/src/Symfony/Component/Cache/Tests/Adapter/FilesystemTest.php
+++ b/src/Symfony/Component/Cache/Tests/Adapter/FilesystemTest.php
@@ -17,7 +17,7 @@ use Symfony\Component\Cache\Adapter\FilesystemAdapter;
 /**
  * @group time-sensitive
  */
-class FilesystemAdapterTest extends CachePoolTest
+class FilesystemTest extends CachePoolTest
 {
     public function createCachePool()
     {
@@ -42,8 +42,8 @@ class FilesystemAdapterTest extends CachePoolTest
             new \RecursiveDirectoryIterator($dir, \RecursiveDirectoryIterator::SKIP_DOTS),
             \RecursiveIteratorIterator::CHILD_FIRST
         );
-        foreach($children as $child) {
-            if ($child->isDir()){
+        foreach ($children as $child) {
+            if ($child->isDir()) {
                 rmdir($child);
             } else {
                 unlink($child);
diff --git a/src/Symfony/Component/Cache/composer.json b/src/Symfony/Component/Cache/composer.json
index ee8dc22..38dc62f 100644
--- a/src/Symfony/Component/Cache/composer.json
+++ b/src/Symfony/Component/Cache/composer.json
@@ -25,7 +25,8 @@
     },
     "require-dev": {
         "cache/integration-tests": "dev-master",
-        "doctrine/cache": "~1.6"
+        "doctrine/cache": "~1.6",
+        "fabpot/php-cs-fixer": "1.11.2"
     },
     "suggest": {
         "symfony/polyfill-apcu": "For using ApcuAdapter on HHVM"
```